lib/engine/game/base.rb

```diff
@@ -1189,6 +1189,14 @@ def legal_tile_rotation?(_entity, _hex, _tile)
         true
       end
 
+      def corporation_parrable?(corporation, parrer)
```

Comment:
can we rename to can_par?(corporation, parrer)

Reply:
I've renamed it